practice-one/styles/base/typography.css

```diff
@@ -0,0 +1,14 @@
+@font-face {
+    font-family: "popins";
+    font-display: swap;
+    src: url("../../assets/fonts/Poppins-ExtraLight.ttf");
+}
+
```

Comment:
Remove 1 redundant line

Reply:
Thank you for your comment. I will fix it